tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks/WriteBuildOutputProps.cs

```diff
@@ -85,6 +84,10 @@ public override bool Execute()
 
                     sw.WriteLine($"    <{propertyName}>{packageIdentity.Version}</{propertyName}>");
                 }
+                foreach (var extraPackage in ExtraPackageIdentities)
+                {
+                    sw.WriteLine($"    <{extraPackage.Id}>{extraPackage.Version}</{extraPackage.Id}>");
```

Comment:
This makes the `ExtraPackageInfo` comment inaccurate, and I think the `ExtraPackageIdentities` property should be cut out too because these aren't package identities anymore. I think renaming `ExtraPackageInfo` to `ExtraProperties`, using it directly, and fixing the comment would be good.

Reply:
Changed as suggested.